### PR TITLE
Some more new struct

### PR DESCRIPTION
**Pull request by HolterPhylo**

### CODE REVIEW

File: libMBIN/Source/NMS/GameComponents/CgExpeditionCategoryStrength.cs [hunk 1]
```diff
@@ -5,8 +5,10 @@
 
 namespace libMBIN.NMS.GameComponents
 {
-    [NMS(Size = 0x14, GUID = 0x3B4F33D91D98F4AF, NameHash = 0xD22200F6DB5FF136, Broken = true)]
+    [NMS(Size = 0x14, GUID = 0x3B4F33D91D98F4AF, NameHash = 0xD22200F6DB5FF136)]
     public class CgExpeditionCategoryStrength : NMSTemplate
     {
+        [NMS(Size = 0x5)]
```
Comment (monkeyman192) [inline libMBIN/Source/NMS/GameComponents/CgExpeditionCategoryStrength.cs]: does this have an associated enum value?

**Comment by monkeyman192** [comment]
This PR doubles up with #206 ...

File: libMBIN/Source/NMS/GameComponents/CgExpeditionCategoryStrength.cs (hunk 1 above)
Comment (HolterPhylo) [reply libMBIN/Source/NMS/GameComponents/CgExpeditionCategoryStrength.cs]: corrected

**Comment by HolterPhylo** [comment]
I don't understand.  #206 and #207 modify different files and you asked me not to make multiple branches from the development branch like I did before? 

Which branch should I make one or many new ones from so has not to have all the changed files in the same "basket"?

**Pull request closed without merging**